scripts/partition_manager/partition_manager.rst

```diff
@@ -174,23 +237,24 @@ span: list OR dict: list
          placement:
             after: [mcuboot]
 
+      # However, the overall configuration is unsatisfiable:
+      # foo requires order 2, while bar requires order 1.
+
       foo:
          span: [mcuboot, app]
 
       bar:
          span: [mcuboot, spm]
 
-      # Order 1: mcuboot, spm, app
-      # Order 2: mcuboot, app, spm
-      # foo requires order 2, while bar requires order 1.
+.. _partition_manager_inside:
 
 inside: list
    This property is the inverse of ``span``.
    The name of the partition that specifies this property is added to the ``span`` list of the first existing container partition in the list.
    This property can be set for image or placeholder partitions.
 
    .. code-block:: yaml
-      :caption: Example for the inside property
+      :caption: Inside property example
```

Comment:
```suggestion
      :caption: "inside" property example
```

Reply:
Why not just stick to the original wording?